src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {

```

Comment:
It's better approach to initialize `ColorSupplier` and `Random` objects inside `Lottery` constructor:
[https://www.baeldung.com/java-variable-instantiation-declaration-vs-constructor](https://www.baeldung.com/java-variable-instantiation-declaration-vs-constructor)

Reply:
Sorry but I have no idea what are you trying to say...